contributors/devel/sig-scheduling/scheduler_benchmarking.md

```diff
@@ -0,0 +1,77 @@
+# Scheduler Benchmarking
+
+Kubernetes scheduler has integration and e2e benchmarks. It is recommended to
+run integration benchmarks at the time of submitting PRs that could potentially
+impact performance of the scheduler and give before and after results with your
+PR submission.
+
+## Running integration benchmarks
+
+To run integration benchmarks use the following command from inside a Kubernetes
+directory. 
+
+```sh
+make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-run=xxx -bench=BenchmarkScheduling"
+```
+
+These benchmarks are located in `./test/integration/scheduler_perf/scheduler_bench_test.go`. 
+The function names start with `BenchmarkScheduling`. At the beginning of each
```

Comment:
I will probably add benchmarks that don't follow this naming scheme. (when I get around to it.)

Reply:
If you plan to add scheduling benchmarks, I think it would make sense if they start with `BenchmarkScheduling`. If they are non-scheduling benchmarks, they should probably be added to other files.

Reply:
If they are non-scheduling benchmarks, then they should be in a different package. We can discuss if I ever get a PR mailed out. :stuck_out_tongue:

Reply:
That's a fair point! :smile: